modules/assignment/src/test/java/com/intuit/wasabi/assignment/impl/AssignmentsImplTest.java

```diff
@@ -346,7 +346,8 @@ public void testGetSingleAssignmentNullAssignmentExperimentExpired() {
         Assignment result = assignmentsImpl.doSingleAssignment(user, appName, label, context, true, true, segmentationProfile, headers);
 
         //Verify result
-        assertThat(result.getStatus(), is(Assignment.Status.EXPERIMENT_EXPIRED));
+        assertThat(result.getStatus(), is(Assignment.Status.EXPERIMENT_PAUSED));
+        verify(experiment).setState(Experiment.State.PAUSED);
```

Comment:
@AndreaSuckro You should also probably verify the call to update the experiment state using the `experimentUtil` object.